modules/administration/pages/ssl-certs-imported.adoc

```diff
@@ -0,0 +1,129 @@
+[[ssl-certs-imported]]
+= Imported SSL Certificates
```

Comment:
Not sure if "Imported SSL Certificates" is correct. 

We want to describe how to "import" SSL Certificates. But the certificate itself is not "imported".
Maybe we talk about just (How to) "Import SSL Certificates" or "Import 3rd patry (signed) SSL Certificates".